src/desktop/apps/fair/fairRedirection.ts

```diff
@@ -0,0 +1,56 @@
+// @ts-ignore
+import { DISABLE_FAIRS_UPDATE_SLUGS, ENABLE_FAIRS_UPDATE } from "../../config"
+
+export const redirectFairRequests = (req, res, next) => {
+  const isFairProfile = res.locals?.profile?.get("owner_type") === "Fair"
```

Comment:
Follow up question (from our grooming). Does this handle the case of 'before the fair has entered the VP preview period'?

As in, the profile either doesn't exist or isn't public. I think it does (there's a check on owner type/exist-y check that implies the profile is public), but wanted to make sure.